src/containers/App.js

```diff
@@ -130,13 +131,18 @@ const App = React.createClass({
             <strong>{contrast.score(colorOne, colorTwo)}</strong> / <strong>{contrast.ratio(colorOne, colorTwo).toFixed(2)}</strong>
           </p>
         </div>
-        <Combination colorOne={colorOne} colorTwo={colorTwo} />
+        <ReactCSSTransitionGroup
+          transitionName="color-switch"
+          transitionEnterTimeout={1300}
```

Comment:
Yeah, I definitely like the "palette cleansing" but it does now feel a touch slow. Perhaps split the difference and do ~750 so it's a little shorter but long enough to cleanse.